lib/minidoc/timestamps.rb

```diff
@@ -15,6 +15,28 @@ def timestamps!
     end
   end
 
+  def set(attributes)
+    if self.class.record_timestamps
+      super(attributes.merge(updated_at: Time.now.utc))
+    else
+      super(attributes)
+    end
+  end
+
+  def atomic_set(query, attributes)
+    if self.class.record_timestamps
+      super(query, attributes.merge(updated_at: Time.now.utc))
+    else
+      super(query, attributes)
+    end
+  end
+
+  def unset(*keys)
+    super
+
+    set(updated_at: Time.now.utc) if self.class.record_timestamps
```

Comment:
Interesting points. The idea of a public `#touch` method is reasonable, but since this would be the only caller right now, and is unlikely to get much use otherwise right now, it doesn't seem worth adding right now.

Personally, I think I'd lean towards either keeping this as-is with the slight redundancy, or doing the slightly opaque `{}` call. WDYT?